packages/web-components/fast-element/src/directives/binding.ts

```diff
@@ -57,6 +57,45 @@ function updatePropertyTarget(this: BindingBehavior, value: unknown): void {
     this.target[this.targetName!] = value;
 }
 
+function updateClassTarget(this: BindingBehavior, value: string): void {
+    const classVersions = this.classVersions || Object.create(null);
+    const target = this.target;
+    let version = this.version || 0;
+
+    // Add the classes, tracking the version at which they were added.
+    if (value !== null && value !== undefined && value.length) {
```

Comment:
I'm curious why you're checking against null and undefined rather than if value is `typeof string`  - is this more performant?

Reply:
That's actually a good question. I'm not sure if a typeof check would be more performant in this case. I can do some experimentation with that and update this in the future.